src/core/audiosource.cpp

```diff
@@ -244,6 +263,14 @@ void FFMS_AudioSource::ResampleAndCache(CacheIterator pos) {
 		OutPlanes, new_req, DecodeFrame->nb_samples,
 		DecodeFrame->extended_data, DecodeFrame->nb_samples * av_get_bytes_per_sample(CodecContext->sample_fmt), DecodeFrame->nb_samples);
 #else
+#if defined(WITH_SWRESAMPLE)
```

Comment:
Again, unnecessary duplication. There should be something like `FFMS_RESAMPLING_ENABLED` that guards the code paths which require either avresample or swresample, and in places like there where there's just a single function call that differs there should be a wrapper function in `ffmscompat.h` that abstracts away the difference.